systems/registry/api-gateway/pkg/rest/api.go

```diff
@@ -5,10 +5,25 @@ package rest
 type GetOrgsRequest struct {
 	UserUUID string `form:"user_uuid" json:"user_uuid" query:"user_uuid" binding:"required" validate:"required"`
 }
+type GetNodeRequest struct {
```

Comment:
We can also add example values on these structs, Like
`example:"brackley-node"`, that way it'll be pre-populated in Postman collection and swagger